admin/kernel-syncdata-service/src/main/resources/application-local.properties

```diff
@@ -177,4 +177,7 @@ syncdata.cache.evict.delta-sync.cron=0 0/15 * * * *
 syncdata.cache.snapshot.cron=0 0 23 * * *
 
 mosip.kernel.auth.appids.realm.map={prereg:'preregistration',ida:'mosip',registrationclient:'mosip',regproc:'mosip',partner:'mosip',resident:'mosip'}
-mosip.kernel.keymanager.unique.identifier.autoupdate=false
\ No newline at end of file
+mosip.kernel.keymanager.unique.identifier.autoupdate=false
+auth.server.admin.issuer.internal.uri=https://iam.dev.mosip.net/auth/realms/
```

Comment:
can you add these in the application-dev.properties as well

Reply:
Done.